app/controllers/api/v2/compute_resources_controller.rb

```diff
@@ -44,7 +44,8 @@ def show
           param :domain, String, :desc => N_("for OpenStack only")
           param :server, String, :desc => N_("for VMware")
           param :set_console_password, :bool, :desc => N_("for Libvirt and VMware only")
-          param :display_type, %w(VNC SPICE), :desc => N_('for Libvirt only')
+          param :display_type, %w(VNC SPICE), :desc => N_('for Libvirt and oVirt only')
+          param :keyboard, Foreman::Model::Ovirt::ALLOWED_KEYBOARD_LAYOUTS, :desc => N_('for oVirt only')
```

Comment:
`Foreman::Model::Ovirt` might actually not be available in a prod environment (it's an optional bundler group). Maybe we can move this to compute resource model.

Reply:
Oh, yeah, didn't think about that one! Will see to move this

Reply:
On the other hand, would it not be presumptuous to say that all keyboard layouts are always the same for all compute resources? And thus, would it not be easier to just set the type to String instead (and let each compute resource for itself decide if the layout is valid?)

Reply:
Without wanting to be overly strict, I think we should make sure the possible values can be found in API docs. But I agree we should not assume the values are the same for all compute resources.

Reply:
Also agreed that the documentation should reflect the possible values. Perhaps it's best to, for now, make the assumption on equality and only _cross the bridge_ when we get to a case in which keyboard layouts are different?

If OK for you, I'll change it to your first proposal :-)

Reply:
Sure, go ahead.